docs/usage/errors.md

```diff
@@ -106,11 +106,16 @@ from pydantic import BaseModel, GetJsonSchemaHandler
 class Model(BaseModel):
     @classmethod
     def __get_pydantic_json_schema__(
-        cls, schema: CoreSchema, handler: GetJsonSchemaHandler
+        cls, core_schema: CoreSchema, handler: GetJsonSchemaHandler
```

Comment:
Nice yeah it’d be a good idea to keep these parameter names standardized in our examples